### ISSUE TITLE
Composing draft outside of N1 results in dozens of drafts inside N1

### ISSUE THREAD

**Issue by hallamoore** [nylas/N1#2724]
Related Zendesk ticket: https://nylas.zendesk.com/agent/tickets/5609

**Comment by emorikawa** [comment]
https://github.com/nylas/N1/issues/2237 may have resurfaced

**Comment by alshedivat** [comment]
Even after deleting all these annoying drafts, the `All Mail` tab still shows these messages as empty. The right panel that is supposed to show the message, just shows the loading animation forever.

![2016-08-16_14-24-59](https://cloud.githubusercontent.com/assets/2126561/17710976/bc464dec-63bd-11e6-8d7c-dc33264bc567.png)

**Comment by danielbayley** [comment]
This also causes N1 to freeze everything when you select that email thread, and drafts are constantly being generated! 😠

**Comment by xclabit** [comment]
This is a big problem and it renders N1 to be useless when it locks up from the massive draft duplications. I'd have to use a different application (OSX Mail.app) to get rid of the drafts. This has been an ongoing problem and it's pretty much turning me away from N1. 

Other people are still posting similar experiences in this 'closed' thread here: https://github.com/nylas/N1/issues/2237#issuecomment-231422304

**Comment by lmerriam** [comment]
I get this all the time still

**Comment by xclabit** [comment]
Yep, I moved on. I haven't seen any activity from developers regarding this.
